app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SessionsController < ApplicationController
+  skip_before_action :require_login, only: [:new,:create]
```

Comment:
Also here, you have to skip more than you need. Better way is to show where you require_login rather where you don't.